src/Components/HugeProject/HugeProject.css

```diff
@@ -0,0 +1,42 @@
+.hugeProjectContainer {
+    align-items: center;
+    background-color: white;
+    border-radius: 20px;
+    box-shadow: 0 0px 20px 0 rgba(79, 73, 63, 0.5);
+    display: flex;
+    flex-direction: column;
+    height: 550px;
+    margin: 10px;
+    width: 550px;
+}
+
+.hugeProjectImg {
+    border-top-left-radius: 20px;
+    border-top-right-radius: 20px;
+    width: 550px;
+    height: 300px;
+}
+
+.hugeProjectTitle {
+    padding: 10px;
+    font-family: "Helvetica Neue", sans-serif;
+    font-size: x-large;
+    font-weight: 600;
+    line-height: 30px;
+    color: #061020;
+}
+
+.hugeProjectDescriptionContainer {
+    display: inline-block;
+    height: min-content;
+    width: 510px;
+    overflow-y: scroll;
+}
+
+.hugeProjectDescription {
+    color: grey;
+    font-size: 18px;
+    font-weight: 500;
+    padding: 0 20px 20px 20px;
+    overflow-y: scroll;
```

Comment:
I tihnk we should bail on the scrolling thing and write shorter descriptions. Then link each project to it's own project subpage where we can flesh them out with images and more text